llama_index/download/download_utils.py

```diff
@@ -26,6 +27,12 @@
 
 PATH_TYPE = Union[str, Path]
 
+LLAMAHUB_ANALYTICS_PROXY_SERVER = "https://llamahub.ai/api/analytics/download"
+class MODULE_TYPE(str, Enum):
+    LOADER = "loader"
+    AGENT = "agent"
```

Comment:
I think technically it's tools instead of agents.

Reply:
changed to tool